kmk/hid.py

```diff
@@ -55,13 +58,60 @@ class HIDUsagePage:
 
 
 class AbstractHID:
-    REPORT_BYTES = 8
+    report_bytes_default = 8
+    REPORT_BYTES = report_bytes_default
+    hid_devices = {}
 
     def __init__(self, **kwargs):
+        self._nkro = False
+        self._mouse = True
+        self._pan = False
+        self.find_devices()
+
+        self._cc_report = bytearray(HID_REPORT_SIZES[HIDReportTypes.CONSUMER] + 1)
+        self._cc_report[0] = HIDReportTypes.CONSUMER
+        self._cc_pending = False
+
+        self.setup_keyboard_hid()
+        self.setup_mouse_hid()
+
+        self.start_watchdog()
 
+    def show_debug(self):
+        if debug.enabled:
```

Comment:
```suggestion
    def show_debug(self):
```
We can save a jump to subroutine if you do:
```
if debug.enabled:
    self.show_debug()
```